CHANGELOG.md

```diff
@@ -86,6 +86,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
   inode numbers consistency after kernel cache flush (not applicable to
   fuse-overlayfs).
 - Remove unwanted colors to STDERR.
+- Remove duplicated group ids.
```

Comment:
Since this is targeted for 1.2.0 please put the log entry under "Other changes" rather than "Bug fixes".

Reply:
refected in https://github.com/apptainer/apptainer/pull/1296/commits/f0529da8a38abcc004052326c3a3596865b8fbb1